libs/react/src/lib/form/textarea/textarea.tsx

```diff
@@ -0,0 +1,80 @@
+import { IExpandableInformation, ILabelAndLabelInformation, IValidator, randomId, validateClassName } from '@sebgroup/extract';
+import { ChangeEvent, memo, useCallback, useEffect, useState } from 'react';
+import { FormItem } from '../../formItem';
+
+export interface ITextAreaProps
+  extends IExpandableInformation,
+    ILabelAndLabelInformation,
+    React.DetailedHTMLProps<React.TextareaHTMLAttributes<HTMLTextAreaElement>, HTMLTextAreaElement> {
+  label: string;
+  labelInformation?: string;
+  validator?: IValidator | undefined;
+  value?: string;
+  'data-testid'?: string;
+}
+
+export const TextArea = memo(
+  ({
+    autoComplete = 'off',
+    expandableInfo,
+    expandableInfoButtonLabel,
+    id = randomId(),
+    label,
+    labelInformation,
+    onChange,
+    role,
+    rows = 4,
+    style = {},
+    validator,
+    value,
+    "data-testid": dataTestId,
+    ...props
+  }: ITextAreaProps) => {
+    const [uuid] = useState(id);
+    const [localValue, setLocalValue] = useState(value);
```

Comment:
I don't know how this works on input for example, but to be able to update what the user writes without getting a new value in through the props, I had to store a "temporary" value. For example in my team we only listen to the onBlur and update the state on that action. So the value props to TextArea is only updated on blur.

Reply:
Yes, this makes sense! I think this needs to be done for the other input fields as well, but that is outside the scope of this PR.